css/style.css

```diff
@@ -195,6 +192,394 @@ body {
   background: 0;
 }
 
+/* Desktop works section start */
+
+.works {
+  display: grid;
+  grid-template: repeat(4, 50px auto auto auto) / repeat(12, 1fr);
+  background: #fff;
+  margin: 20px 118px 130px 118px;
+  padding: 10px;
+  grid-gap: 3px;
+}
+
+#works-title {
+  grid-area: 1 / 1 / 2/ 13;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 2px;
+  flex: 1 1 100%;
+}
+
+#works-title-img {
+  width: 700px;
+  height: auto;
+  padding: 10px 0 10px 10px;
+  flex: 1 1 100%;
+}
+
+.card-1 {
+  grid-area: 2/1/5/13;
+  display: flex;
+  flex-wrap: wrap;
+  flex: 1 1 100%;
+  margin: 0 -5px;
+  padding: 0;
+}
+
+#card-1-img {
+  width: auto;
+  height: auto;
+}
+
+#div-1--right {
+  display: flex;
+  flex-direction: column;
+  width: auto;
+  height: auto;
+  flex: 1;
+  margin: 0 10px 5px 15px;
+  padding: 5px;
+}
+
+.works-title-text {
+  font-family: "Crete Round", sans-serif;
+  font-size: 30px;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  flex: none;
+  margin-bottom: 5px;
+  padding-bottom: 10px;
+  order: 0;
+}
+
+#card-1-text {
+  font-family: "Inter", sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: left;
+  color: black;
+  order: 0;
+  flex: 1 1 100%;
+  margin: 14px 0 5px 5px;
+  padding: 5px;
+}
+
+#skill-text {
+  display: flex;
+  width: 297px;
+  height: 40px;
+  justify-content: start;
+  align-items: center;
+  order: 1;
+  margin: 10px 0;
+  padding: 0;
+}
+
+.about-skills-list > li {
+  text-align: left;
+  margin: 4px;
+}
+
+#skill-text > ul > li {
+  margin: 5px 0;
+  padding: 5px;
+  align-self: center;
+}
+
+.read-text {
+  display: inline-block;
+  width: auto;
+  height: auto;
+  text-decoration: none;
+  padding: 5px;
+  border: 1px solid gray;
+  color: black;
+  order: 1;
+}
+
+.div-1--right-btn {
+  background: #f55800;
+  width: 124px;
+  height: 40px;
+  margin-left: 5px;
+  order: 2;
+  color: #fff;
+  flex: none;
+  cursor: pointer;
+  border: 0;
+}
+
```

Comment:
- [ ] Make sure your button follows the button styling:  **Hover** and **Pressed** check all correct properties assigned as for **color** for Figma Design.

Reply:
Done as advised